nixos/tests/keystone.nix

```diff
@@ -0,0 +1,79 @@
+{ system ? builtins.currentSystem }:
```

Comment:
this test should be referenced in `nixos/release.nix`, otherwise it won't be automatically run.